rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -811,4 +811,11 @@ extern "C" RocalTensor ROCAL_API_CALL rocalSSDRandomCrop(  RocalContext context,
 // /// \param dest_height The output height
 // /// \return Returns a new image that keeps the result.
 
+extern "C" RocalTensor ROCAL_API_CALL rocalPreEmphasisFilter(RocalContext p_context,
+                                                             RocalTensor p_input,
+                                                             RocalTensorOutputType rocal_tensor_output_type,
```

Comment:
In my branches
I have moved rocal_tensor_output_type to the last argument, to make it consistent with image api class, please check and make this change if possible

Reply:
done